apps/postgresql.go

```diff
@@ -79,6 +79,9 @@ func (r MetricsReceiverPostgresql) Pipelines() []otel.ReceiverPipeline {
 				otel.FlattenResourceAttribute("postgresql.index.name", "index"),
 			),
 			otel.MetricsTransform(
+				otel.UpdateMetric("postgresql.bgwriter.duration",
```

Comment:
Is this actually an expected change in this PR?

Reply:
Yes this was a fix for updating to 0.67.0. This change https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/15249 was included in otel 0.63.0 and change `postgresql.bgwriter.duration` data type from int to double, so here we are trying to switch it back to int.